src/python/dbsClient/apis/dbsClient.py

```diff
@@ -165,11 +316,13 @@ def __init__(self, url="", proxy=None, key=None, cert=None, verifypeer=True, deb
         self.key = key
         self.cert = cert
         self.userAgent = userAgent
+        self.accept = accept
+        self.aggregate = aggregate
```

Comment:
Valentin, it's a different idea to trigger this aggregation, but how about defining it something like:
```
self.aggregate = True if accept == "application/json" else False
```
or even:
```
self.aggregate = False if accept == "application/ndjson" else True
```

This way we rely only on the `Accept` header (or option) to define whether data has to be aggregated or streamed, and we could remove the `aggregate` parameter.

Reply:
I might be mixing up the definition of aggregation here.
Is this aggregation that you are implementing meant for:
* keeping the response of the Go and Python servers in exactly the same format
* OR, having the ability to either provide all the result up to the application in a single object, or yield it as retrieved from the server
?

Reply:
What I meant with my second definition is that the client does not need to receive all the data from the server before it starts yielding data to the upstream application. 

In such scenario, the client would yield records as they get to the client, and of course the client needs to know when all the data has been received or not, hence my comment regarding the newline delimited JSON.

This would be the most efficient implementation - in terms of memory footprint both on the server and on the client side - and I thought this is what had been implemented on the Go server, no?

Reply:
yes, this is what `application/ndjson` will provide, and yes this is the most efficient way to handle data stream. Each line is valid JSON, and you don't need list brackets, and comma separator between records. Your client only read data, if newline is present it knows that it is a single JSON record, therefore the entire memory need to parse **ANY** number of records is a size of a single record.

Reply:
Right!

When I read the code you implemented in `parseStream()`, I understand that it would be parsing a response type of `application/ndjson`, which is how DBS2Go server is going to respond to the client calls. In other words, DBS2Go is now streaming:
```
[
JSON record with new line
JSON record with new line
etc
]
```
Is that correct? That method already returns a generator as well, so IF the upstream application is prepared to consume a generator, all the server/client code is already in place.